### PR TITLE
Add interactive URL to response

### PR DESCRIPTION
**Pull request by mryan321**
### What

Adds `url` to response. This is the URL for rendering the interactive through dp-frontend-router to dp-frontend-interactives-controller: https://github.com/ONSdigital/dp-frontend-router/blob/a265f40c9cae102fb92de882eefdf65ca29cb80a/router/router.go#L97

Root domain is driven by config - currently just `SITE_DOMAIN` but I'm thinking we may need a special case for when `IS_PUBLISHING` (i.e. `publishing.develop...`) too - this is a quick fix if required when we come to plugging in preview work.

### How to review

Sanity check enough i think & test cases pass and cover change.

### Who can review

Anyone

### CODE REVIEW

File: features/steps/api_component.go [hunk 1]
```diff
@@ -99,11 +99,9 @@ func NewInteractivesApiComponent(mongoURI string) (*InteractivesApiComponent, er
...
 
 	cfg, _ := config.Get()
-	mongodb := &mongo.Mongo{
-		URI:        strings.Replace(mongoURI, "mongodb://", "", 1),
-		Database:   cfg.MongoConfig.Database,
-		Collection: cfg.MongoConfig.Collection,
-	}
+	cfg.MongoConfig.BindAddr = strings.Replace(mongoURI, "mongodb://", "", 1)
+	cfg.MongoConfig.Database = "interactives-api"
+	mongodb := &mongo.Mongo{Config: cfg}
```
Comment (mwnicks) [inline features/steps/api_component.go:104]: Is this using a real mongo db database for the tests or a mock/stubbed one

Comment (mryan321) [reply features/steps/api_component.go:104]: real 👍 inmem from here: https://github.com/ONSdigital/dp-component-test/blob/f11d339771c967481617d27cb359ba6f9312ccaa/mongo_feature.go#L10 - ref in api: https://github.com/ONSdigital/dp-interactives-api/blob/bc814e66e603005725fc5c2e6e49b2e6f9685e70/main_test.go#L64

Comment (mryan321) [reply features/steps/api_component.go:104]: ideally we should use real everything from docker - i think static files are using similar strategy - im gonna add a ticket for this 😊

Comment (mryan321) [reply features/steps/api_component.go:104]: https://trello.com/c/DyFdTaXn/171-techdebt-better-component-tests-not-using-mocks

Comment (mwnicks) [reply features/steps/api_component.go:104]: Mocked/stubbed 3rd party services is the standard approach in the ONS projects I believe.
I think they are looking at using real services when they start trying to implement integration testing.

**Review by mwnicks — APPROVED** [review]

**Pull request merged**